<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1200</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Highway Signer ‚Äî –£–ø—Ä–∞–≤–ª–µ–Ω–∏–µ –æ–±–º–µ–Ω–æ–º —Ñ–∞–π–ª–∞–º–∏</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="topButtonLayout">
      <item>
       <widget class="QPushButton" name="pushButton_2">
        <property name="maximumSize">
         <size>
          <width>20</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="table_label">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>–†–∞–±–æ—á–∏–µ –º–µ—Å—Ç–∞</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="edit_label">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>–î–µ–π—Å—Ç–≤–∏—è </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="add_btn">
        <property name="toolTip">
         <string>–î–æ–±–∞–≤–∏—Ç—å –Ω–æ–≤–æ–µ —Ä–∞–±–æ—á–µ–µ –º–µ—Å—Ç–æ</string>
        </property>
        <property name="text">
         <string>‚ûï –î–æ–±–∞–≤–∏—Ç—å</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="save_btn">
        <property name="toolTip">
         <string>–°–æ—Ö—Ä–∞–Ω–∏—Ç—å –∏–∑–º–µ–Ω–µ–Ω–∏—è</string>
        </property>
        <property name="text">
         <string>üíæ –°–æ—Ö—Ä–∞–Ω–∏—Ç—å</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="start_stop_btn">
        <property name="toolTip">
         <string>–ó–∞–ø—É—Å—Ç–∏—Ç—å –∏–ª–∏ –æ—Å—Ç–∞–Ω–æ–≤–∏—Ç—å –≤—ã–±—Ä–∞–Ω–Ω–æ–µ —Ä–∞–±–æ—á–µ–µ –º–µ—Å—Ç–æ</string>
        </property>
        <property name="text">
         <string>‚ñ∂ –ó–∞–ø—É—Å—Ç–∏—Ç—å</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="delete_btn">
        <property name="toolTip">
         <string>–£–¥–∞–ª–∏—Ç—å –≤—ã–±—Ä–∞–Ω–Ω–æ–µ —Ä–∞–±–æ—á–µ–µ –º–µ—Å—Ç–æ</string>
        </property>
        <property name="text">
         <string>üóë –£–¥–∞–ª–∏—Ç—å</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="stop_all_btn">
        <property name="toolTip">
         <string>–û—Å—Ç–∞–Ω–æ–≤–∏—Ç—å –≤—Å–µ —Ä–∞–±–æ—á–∏–µ –º–µ—Å—Ç–∞</string>
        </property>
        <property name="text">
         <string>‚èπÔ∏è –û—Å—Ç–∞–Ω–æ–≤–∏—Ç—å –≤—Å–µ</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="maximumSize">
         <size>
          <width>20</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QSplitter" name="main_splitter">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <widget class="QWidget" name="table_container">
       <layout class="QVBoxLayout" name="verticalLayout_table">
        <item>
         <widget class="QTableWidget" name="table">
          <property name="editTriggers">
           <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
          </property>
          <property name="verticalScrollMode">
           <enum>QAbstractItemView::ScrollMode::ScrollPerPixel</enum>
          </property>
          <property name="horizontalScrollMode">
           <enum>QAbstractItemView::ScrollMode::ScrollPerPixel</enum>
          </property>
          <attribute name="horizontalHeaderCascadingSectionResizes">
           <bool>false</bool>
          </attribute>
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>120</number>
          </attribute>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderDefaultSectionSize">
           <number>45</number>
          </attribute>
          <column>
           <property name="text">
            <string>‚Ññ</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Client ID</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>–†–∞–±–æ—á–µ–µ –º–µ—Å—Ç–æ</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>–°—Ç–∞—Ç—É—Å</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>–†–µ–∂–∏–º</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QScrollArea" name="scrollArea">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>789</width>
          <height>352</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_scroll">
         <item>
          <layout class="QGridLayout" name="edit_grid_layout">
           <item row="0" column="0">
            <widget class="QLabel" name="client_id_label">
             <property name="text">
              <string>Client ID:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="client_id_edit"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="workspace_label">
             <property name="text">
              <string>–†–∞–±–æ—á–µ–µ –º–µ—Å—Ç–æ:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="workspace_edit"/>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="host_label">
             <property name="text">
              <string>–•–æ—Å—Ç (server):</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="host_edit"/>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="port_label">
             <property name="text">
              <string>–ü–æ—Ä—Ç:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLineEdit" name="port_edit"/>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="interval_label">
             <property name="text">
              <string>–ò–Ω—Ç–µ—Ä–≤–∞–ª (—Å–µ–∫):</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QLineEdit" name="interval_edit"/>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="incoming_label">
             <property name="text">
              <string>–í—Ö–æ–¥—è—â–∏–µ:</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <layout class="QHBoxLayout" name="incoming_layout">
             <item>
              <widget class="QLineEdit" name="incoming_edit">
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="incoming_btn">
               <property name="text">
                <string>...</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="6" column="0">
            <widget class="QLabel" name="outgoing_label">
             <property name="text">
              <string>–ò—Å—Ö–æ–¥—è—â–∏–µ:</string>
             </property>
            </widget>
           </item>
           <item row="6" column="1">
            <layout class="QHBoxLayout" name="outgoing_layout">
             <item>
              <widget class="QLineEdit" name="outgoing_edit">
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="outgoing_btn">
               <property name="text">
                <string>...</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="7" column="0">
            <widget class="QLabel" name="meta_label">
             <property name="text">
              <string>Meta:</string>
             </property>
            </widget>
           </item>
           <item row="7" column="1">
            <layout class="QHBoxLayout" name="meta_layout">
             <item>
              <widget class="QLineEdit" name="meta_edit">
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="meta_btn">
               <property name="text">
                <string>...</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="8" column="0">
            <widget class="QLabel" name="mode_label">
             <property name="text">
              <string>–†–µ–∂–∏–º —Ä–∞–±–æ—Ç—ã:</string>
             </property>
            </widget>
           </item>
           <item row="8" column="1">
            <layout class="QHBoxLayout" name="mode_layout">
             <item>
              <widget class="QRadioButton" name="mode_client_rb">
               <property name="text">
                <string>–ö–ª–∏–µ–Ω—Ç</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="mode_processor_rb">
               <property name="text">
                <string>–ü—Ä–æ—Ü–µ—Å—Å–æ—Ä</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="mode_client_sign_rb">
               <property name="text">
                <string>–ö–ª–∏–µ–Ω—Ç-–ø–æ–¥–ø–∏—Å—å</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="mode_processor_sign_rb">
               <property name="text">
                <string>–ü—Ä–æ—Ü–µ—Å—Å–æ—Ä-–ø–æ–¥–ø–∏—Å—å</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="9" column="0">
            <widget class="QLabel" name="key_label">
             <property name="text">
              <string>–ü—É—Ç—å –∫ –∫–ª—é—á—É:</string>
             </property>
            </widget>
           </item>
           <item row="9" column="1">
            <layout class="QHBoxLayout" name="key_layout">
             <item>
              <widget class="QLineEdit" name="key_edit">
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="key_btn">
               <property name="text">
                <string>...</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="button_layout">
           <item>
            <widget class="QPushButton" name="generate_key_btn">
             <property name="text">
              <string>–°–≥–µ–Ω–µ—Ä–∏—Ä–æ–≤–∞—Ç—å –∫–ª—é—á</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="log_label">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>–õ–æ–≥–∏:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="log_text">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
